rosserial_tivac/src/rosserial_tivac/make_libraries

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
```

Comment:
Let's not double this up— can you make this script simply call through to the other one?